### PR TITLE
Ask updates

### PR DESCRIPTION
**Pull request by virginiacc**
General Ask updates.

## Additions
- Add sibling page link to English and Spanish answer pages.

## Removals
- Remove HTML editor from answer pages.

## Changes
- Update Spanish mega menu and footer links to replace `Inicio` with `Para consumidores`.
- Display extra sidebar content above disclaimer on answer pages.
- Constrain H2 widths on answer pages.

## Testing

1. Check out branch.
2. Run `python cfgov/manage.py migrate` and `gulp styles`
3. Navigate to [an English Ask answer page](http://localhost:8000/ask-cfpb/can-a-lender-or-dealer-ask-me-about-the-alimony-child-support-or-separate-maintenance-payments-that-i-receive-when-i-apply-for-an-auto-loan-en-1195/) and verify the sidebar links between it and its Spanish sibling 
4. Navigate to [a Spanish page](http://localhost:8000/es/) and verify the updated links in the header/footer
5. Open an [answer page admin edit view](http://localhost:8000/admin/pages/7240/edit/) and verify that the html edit link does not appear in its answer content text editors
6. Open an [answer page admin edit view](http://localhost:8000/admin/pages/7240/edit/). Add a long H2 header in a new table block and verify that the header's width is constrained to the width of the surrounding content.
7. Open an [answer page admin edit view](http://localhost:8000/admin/pages/7240/edit/). Add an email block to its sidebar and verify that the email signup appears between the `About us` and `Legal disclaimer` blocks.

### Spanish answer page with updated menu
Spanish_answer

### English answer page with custom sidebar content
English_page

### CODE REVIEW

**Comment by virginiacc** [comment]
@higs4281 Thanks for reviewing!

> One question: Comprar casa is in the footer nav but not the header. Is that right?

Yes -- it sounds like we don't want to feature `Comprar casa` in the header anymore, but I figured it couldn't hurt to leave it in the footer since there's room & the English version shows different links than the mega menu. Any thoughts, @nataliafitzgerald or @sonnakim?

File: cfgov/unprocessed/css/pages/ask.less [hunk 1]
```diff
@@ -194,16 +199,38 @@
...
+        }
+
+        .about-us-text + .block {
+            margin-top: unit( @grid_gutter-width * 2 / @base-font-size-px, em ) !important;
+        }
+
+        .about-us-text + .disclaimer-text {
+            border-top: 1px solid @block__border;
+            margin-top: unit( @grid_gutter-width * 1.5 / @base-font-size-px, em );
+            padding-top: unit( ( @grid_gutter-width * 1.5 ) / @base-font-size-px, em );
```
Comment (anselmbradford) [inline cfgov/unprocessed/css/pages/ask.less]: VERY TRIVIAL: extra parens in this line versus the line above, but it's the same setting.

Comment (virginiacc) [reply cfgov/unprocessed/css/pages/ask.less]: Thanks for catching that, @anselmbradford -- fixed in 027006168e34fee9ae276dcca21092a4618b7ace

**Comment by sonnakim** [comment]
@virginiacc I think it's ok to leave it - agreed there's space for it, and this takes the same approach as the English version.

**Pull request merged**